pkg/cache/cache_test.go

```diff
@@ -198,6 +197,21 @@ var _ = Describe("Informer Cache", func() {
 				Expect(actual.Namespace).To(Equal(testNamespaceOne))
 			})
 
+			It("should be able to list objects with GVK populated", func() {
+				By("listing pods")
+				listObj := &kcorev1.PodList{}
+				Expect(informerCache.List(context.Background(), listObj)).To(Succeed())
+
+				By("verifying that the returned pods have GVK populated")
+				Expect(listObj.Items).NotTo(BeEmpty())
+				Expect(listObj.Items).Should(HaveLen(3))
+				for _, p := range listObj.Items {
+					Expect(p.GroupVersionKind().Group).To(Equal(kcorev1.SchemeGroupVersion.Group))
+					Expect(p.GroupVersionKind().Version).To(Equal(kcorev1.SchemeGroupVersion.Version))
+					Expect(p.GroupVersionKind().Kind).To(Equal("Pod"))
```

Comment:
`Expect(p.GroupVersionKind()).To(Equal(kcorev1.SchemeGroupVersion.WithKind("Pod")))`

Reply:
Done.